.travis.yml

```diff
@@ -16,11 +16,11 @@ matrix:
     - os: osx
       osx_image: xcode7.3
       language: generic
-      env: TRAVIS_PYTHON_VERSION="2"
+      env: TRAVIS_PYTHON_VERSION="2" BREW_DEPS="cmake\nhdf5\nboost\nlibjpeg\nlibtiff\nlibpng\nfftw\nreadline\npython"
     - os: osx
       osx_image: xcode7.3
       language: generic
-      env: TRAVIS_PYTHON_VERSION="3"
+      env: TRAVIS_PYTHON_VERSION="3" BREW_DEPS="cmake\nhdf5\nboost\nlibjpeg\nlibtiff\nlibpng\nfftw\nreadline\npython3"
```

Comment:
Make brew dependencies compact so that we can easily parse them for their dependencies. Also, this allows for us to do installation in one pass.